### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by lgeiger**
- Remove `npm` from dependencies. Instead specify Atom version.
- Add images for code completion and introspection.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,7 +81,7 @@ Tested and works with:
 - [jupyter-nodejs](https://github.com/notablemind/jupyter-nodejs)
 - [IRkernel](https://github.com/IRkernel/IRkernel) (install the "Development" version from `master` — necessary changes haven't gotten released as binaries yet)
 
-But it _should_ work with any kernel — [post an issue](https://github.com/willwhitney/hydrogen/issues) if anything is broken!
+But it _should_ work with any [kernel](https://github.com/ipython/ipython/wiki/IPython-kernels-for-other-languages) — [post an issue](https://github.com/willwhitney/hydrogen/issues) if anything is broken!
```
Comment (rgbkrk) [inline README.md]: :+1:

File: README.md [hunk 2]
```diff
@@ -24,7 +25,7 @@ Hydrogen was inspired by Bret Victor's ideas about the power of instantaneous fe
 
 For all systems, you'll need
 
-- [`npm`](https://docs.npmjs.com/getting-started/installing-node)
+- [Atom](https://atom.io/) `1.6.0+`
```
Comment (rgbkrk) [inline README.md]: Thank you

File: README.md [hunk 3]
```diff
@@ -128,7 +129,17 @@ You can re-run the watch expressions by using the normal run shortcut (⌘-⌥-
...
 
+Receive completions from the running kernel.
+
+<img width="416" src="https://cloud.githubusercontent.com/assets/13285808/14108987/35d17fae-f5c0-11e5-9c0b-ee899387f4d9.png">
+
+### Code Introspection
+
+You can use the "Hydrogen: Inspect" command from the Command Palette to get metadata from the kernel about the object under the cursor.
+
+<img width="770" src="https://cloud.githubusercontent.com/assets/13285808/14108719/d72762bc-f5be-11e5-8188-32725e3d2726.png">
```
Comment (rgbkrk) [inline README.md]: Solid demo images here!

**Comment by rgbkrk** [comment]
Your new avatar threw me off! I thought we had a new collaborator. :smile: Great to see you're a living person!

**Comment by lgeiger** [comment]
Yes, the avatar was overdue. :smile:

**Pull request merged**